content/v2.1/helpers/form_helper.md

```diff
@@ -0,0 +1,964 @@
+---
+title: Form Helper
+order: 30
+---
+# Form Helper
+
+Yields a form builder for constructing an HTML form and returns the resulting form string.
+
+Please read this section for the methods for building the form's fields.
+
+## form_for
+
+Builds the form using the given base name for all fields.
+The overall structure of the attributes available is as follows:
+
+### Usage
+
+```ruby
+form_for(base_name, url, values: _form_for_values, params: _form_for_params, **attributes)
+```
+
+**Basic Example**
+
+```ruby
+<%= form_for("book", "/books", class: "form-horizontal") do |f| %>
+  <div>
+    <%= f.label "title" %>
+    <%= f.text_field "title", class: "form-control" %>
+  </div>
+
+  <%= f.submit "Create" %>
+<% end %>
+```
+
+This will render HTML as shown below:
+
+```html
+<form action="/books" method="POST" accept-charset="utf-8" class="form-horizontal">
+  <input type="hidden" name="_csrf_token" value="920cd5bfaecc6e58368950e790f2f7b4e5561eeeab230aa1b7de1b1f40ea7d5d">
+  <div>
+    <label for="book-title">Title</label>
+    <input type="text" name="book[title]" id="book-title" value="Test Driven Development">
+  </div>
+
+  <button type="submit">Create</button>
+</form>
+```
+
+As you can see, all the fields had been prefixed with the form's name, which is a string `book`, passed as a first argument.
+
+To show CSRF meta tags, you need to have [sessions enabled](/v2.0/actions/sessions). Without that, you won't have the `csrf_token` hidden input field available.
```

Comment:
```suggestion
```

@swilgosz This is out of context.
You wrote a section that explains CSRF protection, maybe let's move it there.

Reply:
I disagree. Because we have sessions disabled by default, but our examples assume it's enabled, I think it makes sense to mention the chapter when it's explained how to enable sessions. I've just removed the CSRF explanation, but kept the link